### PR TITLE
[AM][OPS-10906] dependency updates

### PR DESCRIPTION
**Pull request by almorcrette**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -7,14 +7,14 @@ object AppDependencies {
   val compile = Seq(
     ws,
     "uk.gov.hmrc"                   %% "bootstrap-backend-play-28"  % boostrapVersion,
-    "com.fasterxml.jackson.module"  %% "jackson-module-scala"       % "2.14.2"
+    "com.fasterxml.jackson.module"  %% "jackson-module-scala"       % "2.15.2"
```
Comment (AndyHWChung) [inline project/AppDependencies.scala:10]: what happens if you now remove this dependency - does the service still work?

Comment (almorcrette) [reply project/AppDependencies.scala:10]: Then we get the vulnerable library back in use

Comment (almorcrette) [reply project/AppDependencies.scala:10]: I did look at taking it out, but we don't want to

Comment (almorcrette) [reply project/AppDependencies.scala:10]: Otherwise it compiles and tests pass

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**